apis/r/tests/testthat/test-shape.R

```diff
@@ -202,7 +202,7 @@ test_that("SOMADataFrame shape", {
             expect_equal(str_dom, c("", ""))
           } else {
             if (is.null(str_dfc)) {
-                expect_equal(str_dom, c("", ""))
+              expect_equal(str_dom, c("", ""))
             } else {
               expect_equal(str_dom, str_dfc)
             }
```

Comment:
This whole chunk could be simplified to
```r
expect_equal(str_dom, str_dfc %||% c("", ""))
```
or optionally with an info message telling us the `str_dfc` if this test fails
```r
expect_equal(str_dom, str_dfc %||% c("", ""), info = sprintf("str_dfc: (%s)", paste(str_dfc %||% "NULL", collapse = ", ")))
```

Reply:
Yikes! As a non-R-expert writer (and with an eye to non-R-expert readers) I'm inclined to keep it the way it is, but, I very much appreciate the pro-tip! :)